.env

```diff
@@ -0,0 +1,16 @@
+ES_SEARCH_HOSTNAME=es-search
```

Comment:
I don't think we should push this file to source control. It should be pinned to our slack channel however.

Reply:
Normally, I would agree, but none of these values are secret; all of this information is available in the source code we already have pushed. That’s why I made the call to check this file in.

Reply:
I could call this file ‘.env.example’ or something, but it’s 100% usable as-is, so it’s not really an example. It’s the same file that you would download from Slack. My argument is that since none of these values are secret, let alone custom from developer-to-developer, we might as well just leave it in source control and skip the extra hoop of having to download the file on every fresh clone.

Reply:
Alternatively, I can add _all_ of the `.env` values as defaults in the source code, and then the application won’t even need to have a `.env` to run. This would mean we’re hard-coding our development values into our source code, which we’re most of the way to anyways. I don’t particularly like that idea, but I also don’t particularly like checking in the `.env` file. @tnoelcke thoughts?

Reply:
That works for me. Less chance of people forgetting that way.